api/swagger.yml

```diff
@@ -1581,6 +1581,56 @@ components:
         - completed
         - update_time
 
+    MergeAsyncStatus:
+      type: object
+      properties:
+        task_id:
+          type: string
+          description: the id of the async merge task
+        completed:
+          type: boolean
+          description: true if the task has completed (either successfully or with an error)
+        update_time:
+          type: string
+          format: date-time
+          description: last time the task status was updated
+        result:
+          $ref: "#/components/schemas/MergeResult"
+        error:
+          $ref: "#/components/schemas/Error"
+        status_code:
+          type: integer
+          description: the status code of the error if it exists
+      required:
+        - task_id
+        - completed
+        - update_time
+
+    CommitAsyncStatus:
```

Comment:
Except result everything here seems to be duplicate of the merge status. Consider consolidating these in a single schema and sharing it among the two